frontend/src/libs/components/link/styles.module.scss

```diff
@@ -0,0 +1,8 @@
+.link {
```

Comment:
We have some font properties that we need to set too, to make it look like on design. Pay attention to the text block when you select Sign In in figma. So you also need to:

1. setup custom fonts
 - download Monserat fonts and add them to public/fonts folder
 - create fonts.scss in assets/css/ and define via font-face
 2. add font, font-size, font-weight, line-height properties to the link styles with values from figma design

<img width="540" alt="Screenshot 2023-08-20 at 19 39 20" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/64521658/c90b4d0a-bf8f-4971-8c79-a59fc48f0e1d">

Reply:
The official docs have font-face definitions: https://github.com/JulietaUla/Montserrat/tree/master/fonts/webfonts I use Montserrat.css

Reply:
I modified the css to keep only the normal fonts with woff2 format (the lightest one).

Reply:
solved